front_end_pocket_poll/lib/groups_widgets/group_page.dart

```diff
@@ -86,6 +86,14 @@ class _GroupPageState extends State<GroupPage>
     EventsList.eventsTypeOccurring: null
   };
 
```

Comment:
Could you add some docs here just to drive home what this does even further?

Reply:
added doc